app/helpers/job_invocation_output_helper.rb

```diff
@@ -0,0 +1,32 @@
+module JobInvocationOutputHelper
+  def colorize_line(line)
+    line = line.gsub(/\e\[.*?m/) { |seq| color = seq[/(\d+)m/,1]; "{{{format color:#{color}}}}" }
+
+    @current_color ||= 'default'
+    out = %{<span style="color: #{@current_color}">}
+    parts = line.split(/({{{format.*?}}})/)
+    parts.each do |line|
+      if line.include?('{{{format')
+        if color_index = line[/color:(\d+)/, 1]
+          @current_color = case color_index
+                           when '31'
+                             'red'
+                           when '32'
+                             'green'
+                           when '33'
+                             'orange'
+                           when '0'
```

Comment:
When testing with `ls -al --color=always`, the lightblue for code 34 looked very nice and I suggest to include that as well
![res-ansi-2](https://cloud.githubusercontent.com/assets/190443/22526808/f69564a8-e8cc-11e6-8cba-318b24e369ea.png)